src/admin-views/editor/panel/ticket.php

```diff
@@ -249,7 +522,21 @@ class="button-primary tribe-dependent tribe-validation-submit"
 					id="rsvp_form_save"
 					class="button-primary tribe-dependent tribe-validation-submit"
 					name="ticket_form_save"
-					value="<?php echo esc_attr( sprintf( _x( 'Save %s', 'RSVP form save value', 'event-tickets' ), tribe_get_rsvp_label_singular( 'form_save_value' ) ) ); ?>"
+					value="
```

Comment:
Needs one-liner treatment (define string at top, echo and escape on one line), specifically no extra whitespace in the attribute quotes.

Reply:
done